lib/hammer_cli/ssloptions.rb

```diff
@@ -0,0 +1,27 @@
+module HammerCLI
+  def self.get_ssl_options
+    settings = HammerCLI::Settings
+    logger = Logging.logger['SSLoptions']
+    ssl_options = {}
+    for sslopt in [:ssl_ca_file, :ssl_ca_path] do
+      ssloptval = settings.get(:_params, sslopt) || settings.get(:ssl, sslopt)
+      ssl_options[sslopt] = ssloptval if ssloptval
+    end
+    ssl_client_cert = settings.get(:_params, :ssl_client_cert) || settings.get(:ssl, :ssl_client_cert)
+    ssl_client_key = settings.get(:_params, :ssl_client_key) || settings.get(:ssl, :ssl_client_key)
+    if ssl_client_cert && ssl_client_key
+      ssl_options[:ssl_client_cert] = OpenSSL::X509::Certificate.new(File.read(ssl_client_cert))
+      ssl_options[:ssl_client_key] = OpenSSL::PKey::RSA.new(File.read(ssl_client_key))
+    elsif ssl_client_cert
+      logger.warn("SSL client certificate is set but the key is not, SSL client authentication disabled")
```

Comment:
I think this message is important enough to display it to users as a standard warning:
warn _("SSL client...")